docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-cli-example-create.md

```diff
@@ -213,6 +213,141 @@ Name              Tier        Version        State     Health    Regions
 my-multi-region   Dedicated   2.14.7.0-b51   ACTIVE    💚        us-central1,+2   3         6 / 24GB / 600GB
 ```
 
+## Create, update, and describe an EAR-enabled cluster
+
+### Create an EAR-enabled cluster
+
+Use the following commands to create a new cluster with Encryption at Rest (EAR) enabled using a customer-managed key (CMK) in AWS, GCP, or Azure.
+
+AWS:
```

Comment:
Use tabs - see the Prerequisites link for example

Reply:
I used tabs, and when I was switching tabs, it didn't reflect the data, and it remained the same with one of the tab data; maybe you can take a look once I update the PR.